drgn/cli.py

```diff
@@ -258,7 +273,16 @@ def _main() -> None:
     else:
         logger.setLevel(args.log_level.upper())
 
-    prog = drgn.Program()
+    platform = None
+    if args.architecture:
+        platform = drgn.Platform(drgn.Architecture[args.architecture.upper()])
+
+    vmcoreinfo = None
+    if args.vmcoreinfo:
```

Comment:
Nit:
```suggestion
    if args.vmcoreinfo is not None:
```
(Not that an empty path is particularly meaningful, but I'd prefer that we get an error for that instead of silently ignoring it.)